keps/sig-apps/3939-allow-recreation-when-fully-terminated/README.md

```diff
@@ -0,0 +1,699 @@
+
+We will update the Status field in the same place as when we update the number of active jobs so it should not require any extra API calls.  
+
+In cases where pods are terminating outside of the job controller, we will look at the list of pods that a job has.  And we can classify them as terminating and add that to the status field.
+
+### Test Plan
+
+<!--
+**Note:** *Not required until targeted at a release.*
+The goal is to ensure that we don't accept enhancements with inadequate testing.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
+when drafting this test plan.
+
+[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
+-->
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+#### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+- `controller_utils`: `April 3rd 2023` - `56.6`
+- `job`: `April 3rd 2023` - `90.4`
+
+##### Integration tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+- <test>: <link to test coverage>
+-->
+
+We will add the following integration test for the Job controller:
+
+JobRecreatePodsWhenFailed Feature Toggle On:
+
+  1) Job starts pods that takes a while to terminate
+  2) Delete pods
+  3) Verify that pod creation only occurs once terminating pods are removed
+
+We should test the above with the FeatureToggle off also.
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+- <test>: <link to test coverage>
+-->
+
+### Graduation Criteria
+
+#### Alpha
+
+- Job controller can consider terminating pods as active
+- Job controller counts terminating pods in `JobStatus`.
+- Unit Tests
+- Integration tests
+
+#### Beta
+
+- Address reviews and bug reports from Alpha users
+- E2e tests are in Testgrid and linked in KEP
+- The feature flag enabled by default
+
+#### GA
+
+- Address reviews and bug reports from Beta users
+- Write a blog post about the feature
```

Comment:
That is not a hard requirement, I wouldn't place it here explicitly.

Reply:
Are you saying to remove the entire graduate criteria section?  Or just the mention of the Blog Post?

Reply:
the mention of the blogpost